playbooks/portals-setup-following.yml

```diff
@@ -105,10 +105,10 @@
       include_tasks: tasks/portal-get-log-filename-prefix.yml
 
     - name: Include getting Ansible repo branch and commit
-      include_tasks: tasks/playbook-version-get.yml
+      include_tasks: tasks/portal-version-get.yml
```

Comment:
We are not getting a version of portal, but of ansible playbook.
Maybe we can try to rename it to something like: `ansible-playbooks-version-get.yml` or `ansible-repo-version-get.yml`.